<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clrplt</class>
 <widget class="QMainWindow" name="clrplt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>clrplt</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_file"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionblack_gray_white"/>
    <addaction name="actionblue_white_red"/>
    <addaction name="actiongreen_yellow_red"/>
    <addaction name="actionColor_scaling"/>
    <addaction name="actionGrid"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionColorPlot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMap"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_file">
   <property name="text">
    <string>Add file</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionblack_gray_white">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>black_gray_white</string>
   </property>
  </action>
  <action name="actionblue_white_red">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>blue_white_red</string>
   </property>
  </action>
  <action name="actiongreen_yellow_red">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>green_yellow_red</string>
   </property>
  </action>
  <action name="actionColor_scaling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Color scaling</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="actionColorPlot">
   <property name="text">
    <string>ColorPlot</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
